packages/shared/forks/ReactFeatureFlags.www-dynamic.js

```diff
@@ -24,6 +24,7 @@ export const enableUnifiedSyncLane = __VARIANT__;
 export const enableTransitionTracing = __VARIANT__;
 export const enableDeferRootSchedulingToMicrotask = __VARIANT__;
 export const alwaysThrottleRetries = __VARIANT__;
+export const throttleDisappearingFallbacks = __VARIANT__;
```

Comment:
I usually try to prefix the flag with `enable` so it's clear which direction the feature is going (`always` works too, as a mirror of the other flag)